### ISSUE TITLE
downsize supporters' avatars to reasonable size

### ISSUE THREAD

**Issue by dym-sh** [withastro/astro.build#9]
atm some pfps are unfiltered unprocessed uploads, with sizes reaching 2500px² at 2.3 MB — all just to display as a 50px² thumbnail

e.g. `https://opencollective-production.s3.us-west-1.amazonaws.com/85bf0a60-0e13-11ec-9e4c-61b71d0aff58.jpeg`

![fullsize-avatars](https://user-images.githubusercontent.com/59403389/149653095-3c96c366-b309-4181-bcef-2dac19284685.png)

**Comment by natemoo-re** [comment]
Yeah I would love to tackle this, but we're currently just pulling them in directly from Open Collective. Would need to add a script to download them at build time and resize the images appropriately.

Open to contributions! Just not sure I have time for this at the moment.

**Comment by dym-sh** [comment]
is there not an image proxy for general purposes within astro?

**Comment by jgil-r** [comment]
> Yeah I would love to tackle this, but we're currently just pulling them in directly from Open Collective. Would need to add a script to download them at build time and resize the images appropriately.
> 
> Open to contributions! Just not sure I have time for this at the moment.

What would you use to resize the images?

**Comment by dym-sh** [comment]
https://docs.netlify.com/routing/redirects/rewrites-proxies/
+ https://docs.netlify.com/large-media/transform-images/

**Comment by jasikpark** [comment]
https://www.zachleat.com/web/indieweb-avatar/ 👀
